CMakeLists.txt

```diff
@@ -33,13 +33,15 @@ MESSAGE(STATUS "Build-Specific Libraries: ${BUILD_SPECIFIC_LIBRARIES}")
 
 SET(EXECUTABLE_OUTPUT_PATH ${PROJECT_SOURCE_DIR}/bin)
 SET(LIBRARY_OUTPUT_PATH ${PROJECT_SOURCE_DIR}/lib)
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/submodules")
+SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/submodules")
 
 INCLUDE_DIRECTORIES(${PROJECT_BINARY_DIR})
 INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR})
 INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR}/src)
 INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR}/src/config_reader/include)
-INCLUDE_DIRECTORIES(/usr/include/spinnaker)
+INCLUDE_DIRECTORIES(/opt/spinnaker/include)
```

Comment:
Looks like this may be system dependent - this directory does not exist on my computer, instead it is at `/usr/include/spinnaker`.

I recommend you use the CMake search procedure, and provide both these as options for the search.

Reply:
On both of my 20 installs it installs to /opt, but I will find a way to make this non system dependent